src/main/java/ru/sbtqa/tag/pagefactory/Page.java

```diff
@@ -808,6 +808,11 @@ public String getElementTitle(WebElement element) {
         for (Map.Entry<Field, String> entry : PageFactory.getPageRepository().get(this.getClass()).entrySet()) {
             try {
                 if (Core.getElementByField(this, entry.getKey()) == element) {
+                    if (entry.getKey().getAnnotation(ElementTitle.class) != null
```

Comment:
Should we extract entry.getKey().getAnnotation(ElementTitle.class) and entry.getKey().getAnnotation(ElementTitle.class).value() into local variables ?

Reply:
I don't think that it's good idea. Now it's look more elegantly IMHO

Reply:
Fixed in https://github.com/sbtqa/page-factory/pull/112